src/main/java/edu/cornell/kfs/pmw/batch/PaymentWorksConstants.java

```diff
@@ -104,6 +105,19 @@
         public static final String NEW_VENDOR_REQUESTS_PAYEE_ACH_REPORT_NAME = "New Vendor Requests Create KFS Payee ACH Account";
     }
     
+    public static final class PaymentWorksCredentialKeys {
+        public static final String PAYMENTWORKS_API_URL = "paymentworks.api.url";
+        public static final String PAYMENTWORKS_USER_ID = "paymentworks.user.id";
+        public static final String PAYMENTWORKS_AUTHORIZATION_TOKEN = "paymentworks.authorization.token";
+    }
```

Comment:
Wouldn't PaymentWorksWebServiceConstants be a better place to have this class defined as these items pertain to that functionality?